wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,74 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly find the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you need drivers for and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you need drivers for.
+
+## Method 1: Check Windows Update
+Most of the time, you will find updates for your driver.
+
+ 1. Open the configuration for Windows via windows menu or the searcher.
+ 2. Go to Updates and security.
+ 3. Click to “Check updates”.
+ 4. Windows Update might say that you are up-to-date even if optional updates are available. In the case you see optional updates, and you click into it, you will see some updates (normally drivers), you select to all, and you need to click “Download and install” button.
+
+## Method 2: Download via brand device page.
+It's highly recommended to downloading the drivers or update them via brand page or app by the fact that the company verifies that the driver works well with the computer (graphics in most cases is on the brand GPU page)
+All you need to do is:
+
+ 1. Go to your favourite searcher and search: “{your brand device name} drivers”.
+ 2. Next, put the ID of your device and follow the instructions on the page.
+ 3. When you selected the product you have two options:
+	- Download the driver downloader (recomended)
+	- And download the driver on especific and run executable
+
+## Method 3: Download via brand product page.
+Now if you want to download for your GPU or your brand device hasn't the drivers (in this case do it if it is especially important, otherwise apply the saying: “if it works, don't touch it, or you may make it worse”).
+Now follow the same instructions but the brand hardware (like intel CPU and GPU).
+
+ 1. Go to your favourite searcher again and search: “{your brand name} driver download”.
+ 2. Search your hardware piece you want to upgrade.
+ 3. When you selected the product you have two options:
+	- Download the driver downloader (recomended)
+	- And download the driver on especific and run executable
+ - (Follow method 3 for more information: https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-2-device-manager)
```

Comment:
```suggestion
 - (Follow method 3 for more information: [https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website))
```
You may want to properly link to Method 3 since that is what you're referencing, and the link should be clickable.